src/metricCollector.ts

```diff
@@ -42,9 +44,11 @@ export class MetricCollector {
     opts: MetricCollectorOptions,
     registers: Registry[] = [globalRegister],
   ) {
-    const { logger, autoDiscover, redis, metricPrefix, ...bullOpts } = opts;
+    const { logger, autoDiscover, redis, redisPassword, metricPrefix, ...bullOpts } = opts;
+    logger.info('connection to redis ' + redis + ' with password ' + redisPassword);
```

Comment:
it's a shame you log the password there... I had to make a fork just to remove it and re-publish

Reply:
Used for debugging and forgot to remove it.

thanks @maxaggedon